recipe/meta.yaml

```diff
@@ -11,22 +11,23 @@ source:
   patches:
     # http://lists.gnu.org/archive/html/bug-ncurses/2011-04/txtkWQqiQvcZe.txt
     - fix.patch
+    - ncurses-5.9-gcc-5.patch
 
 build:
   skip: true  # [win]
-  number: 0
+  number: 1
   detect_binary_files_with_prefix: true
 
 test:
   files:
     - run_test.sh         # [osx]
     - run_test.py         # [osx]
   commands:
-    - exit $(test -f ${PREFIX}/lib/libncurses.a)
-    - exit $(test -f ${PREFIX}/lib/libtinfo.a)
-    - exit $(test -f ${PREFIX}/lib/libform.a)
-    - exit $(test -f ${PREFIX}/lib/libmenu.a)
-    - exit $(test -f ${PREFIX}/lib/libpanel.a)
+    - exit $(test -f ${PREFIX}/lib/libncursesw.a)
+    - exit $(test -f ${PREFIX}/lib/libtinfow.a)
+    - exit $(test -f ${PREFIX}/lib/libformw.a)
+    - exit $(test -f ${PREFIX}/lib/libmenuw.a)
+    - exit $(test -f ${PREFIX}/lib/libpanelw.a)
```

Comment:
So, not that this was your error (definitely mine), but would you mind stripping `exit $(` and `)` from the ends of these tests?